linebot/models/responses.py

```diff
@@ -473,3 +473,35 @@ def __init__(self, request_id=None, **kwargs):
         super(NarrowcastResponse, self).__init__(**kwargs)
 
         self.request_id = request_id
+
+
+class AudienceGroup(Base):
+    """AudienceGroups.
+
+    https://developers.line.biz/en/reference/messaging-api/#get-audience-group
+    """
+
+    def __init__(self, audience_group_id=None, type=None, description=None, status=None,
+                 audience_count=None, created=None, is_ifa_audience=None,
+                 permission=None, create_route=None, request_id=None, **kwargs):
+        """__init__ method.
+
+        :param audienceGroups: List of audience groups.
```

Comment:
[nits]
Please update the doc as reference.
https://developers.line.biz/en/reference/messaging-api/#response-26

Reply:
Completed